src/imageio/format/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ cmake_minimum_required(VERSION 2.6)
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/../../" "${CMAKE_CURRENT_SOURCE_DIR}")
 include_directories(SYSTEM "${PNG_PNG_INCLUDE_DIR}")
 
+set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+
+add_definitions(-include modules_api.h)
```

Comment:
I guess that should have some path, like the next one, and like we use `#include` in the code?